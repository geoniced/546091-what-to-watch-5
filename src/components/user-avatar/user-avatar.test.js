import React from "react";
import {BrowserRouter} from "react-router-dom";
import renderer from "react-test-renderer";
import UserAvatar from "./user-avatar";

it(`renders UserAvatar component`, () => {
  const tree = renderer
    .create(
        <BrowserRouter>
          <UserAvatar />
        </BrowserRouter>
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
