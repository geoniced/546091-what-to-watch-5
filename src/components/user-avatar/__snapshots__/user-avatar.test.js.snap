// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders UserAvatar component 1`] = `
<a
  href="/mylist"
  onClick={[Function]}
>
  <div
    className="user-block__avatar"
  >
    <img
      alt="User avatar"
      height="63"
      src="img/avatar.jpg"
      width="63"
    />
  </div>
</a>
`;
